<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolForGitCommit</class>
 <widget class="QWidget" name="ToolForGitCommit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QPushButton" name="m_btnAdd">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="text">
      <string>Feature Format</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnModify">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="text">
      <string>Pronto Format</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnDel">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="text">
      <string>Common Format</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
